meson.build

```diff
@@ -59,7 +59,15 @@ endif
 add_project_arguments(vkd3d_compiler.get_supported_arguments([
     '-Wno-format',
     '-Wno-missing-field-initializers',
-    '-Wno-unused-parameter']),
+    '-Wno-unused-parameter',
+    '/wd4244', # Narrowing conversion
+    '/wd4101', # Unreferenced local variable
+    '/wd4267', # Another narrowing conversion
+    '/wd4996', # Secure no warnings
+    '/wd4334', # Result of 32-bit shift cast to 64-bit
+    '/wd4146', # Unary minus on unsigned
+    '/wd4305', # Truncation from double to float
+    ]),
```

Comment:
Do we pass all of these to gcc as well and does that work?

Reply:
It's filtered by Meson based on what works.

Reply:
Similarly, GNU-style flags are filtered out when using MSVC.

Reply:
Ah, nice.